### PR TITLE
String tiers translate filter

### PR DESCRIPTION
**Pull request by phlax**
Adds tag filters to translate filtering dropdown

### CODE REVIEW

File: pontoon/base/models.py [hunk 1]
```diff
@@ -2216,8 +2216,11 @@ def for_project_locale(
                 )
             )
 
+        if tag:
+            post_filters.append(Q(resource__tag__slug__in=tag.split(',')))
+
         if post_filters:
-            entities = entities.filter(Q(*post_filters))
+            entities = entities.filter(Q(*post_filters)).distinct()
```
Comment (phlax) [inline pontoon/base/models.py]: `distinct` is only needed if tag is set

File: pontoon/base/views.py [hunk 2]
```diff
@@ -102,6 +103,7 @@ def translate(request, locale, slug, part):
         'part': part,
         'project': project,
         'projects': projects,
+        'tags': TagsTool(projects=[project])
```
Comment (phlax) [inline pontoon/base/views.py]: this requires `priority=True`

File: pontoon/base/views.py [hunk 3]
```diff
@@ -102,6 +103,7 @@ def translate(request, locale, slug, part):
         'part': part,
         'project': project,
         'projects': projects,
+        'tags': TagsTool(projects=[project], priority=True)
```
Comment (phlax) [inline pontoon/base/views.py]: add `if project.tags_enabled`

**Comment by phlax** [comment]
TODO:

- make links in dashboards work with tag filtering
- single click filtering doesnt work atm
- update icons for tags in filter dropdown

**Comment by mathjazz** [comment]
For the icon I suggest we use the [Font Awesome](https://fontawesome.com/cheatsheet) "tag" icon: .

**Comment by mathjazz** [comment]
Instead of using the (translation) `status` parameter, we should introduce the `tag` parameter and use that for filtering by tag.

**Comment by phlax** [comment]
> Instead of using the (translation) status parameter,

this was the bug "single click filtering doesnt work atm"

File: pontoon/base/static/css/translate.css [hunk 4]
```diff
@@ -708,6 +708,12 @@ body > header aside p {
   display: none;
 }
 
+[class^="tag-"] .status.fa:before,
+[class*=" tag-"] .status.fa:before {
```
[the remark this answers is not in the document]
Comment (phlax) [reply pontoon/base/static/css/translate.css]: either should work - but the current selector is a bit more specific/accurate

**Pull request merged**